.github/issue_template.md

```diff
@@ -0,0 +1,46 @@
+<!-- Thanks for the interest in Spring Cloud Data Flow! Before you create a new issue, please answer the following question.-->
+
+**Is this a "Bug Report" or "Feature Request"?** _(choose one and remove the irrelevant sections before submitting the issue)_:
+
+<!--
+If this is a "Bug Report", please:
+
+**Describe**
+A clear and concise description of what the bug is. Additionally, it'd help if you can include the logs and entire stacktrace including the "caused by". _(see [GitHub Mardown](https://guides.github.com/features/mastering-markdown/) for logs/code formatting guidelines)_
+
+**Release versions**
+There's an [API](http://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#api-guide-resources-server-meta-retrieving) to gather SCDF's system information including the dependent projects and the associated versions. Alternatively, you can capture this information from the Dashboard's [About](http://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#dashboard) tab, so make sure to include the copied JSON in the bug report.
+
+**Custom Apps**
+If your Stream or Task data pipeline include custom apps and there's a problem associated with it, it'd be good to review the sample App and the release versions in use. Make sure to include the register, create, and deploy/launch DSL commands for completeness.
```

Comment:
Do we want to mention that'd it help if they could share a repo with a sample of the app that exhibits the problem?

Reply:
That's what I meant by "it'd be good to review the sample App and the release versions in use" - I can clarify more.